apps/openmw/mwmechanics/combat.cpp

```diff
@@ -75,10 +75,11 @@ namespace MWMechanics
         if (!blocker.getRefData().getBaseNode())
             return false; // shouldn't happen
 
+        assert(!blocker.getRefData().isBaseNodeFlatten());
         float angleDegrees = osg::RadiansToDegrees(
                     signedAngleRadians (
                     (attacker.getRefData().getPosition().asVec3() - blocker.getRefData().getPosition().asVec3()),
-                    blocker.getRefData().getBaseNode()->getAttitude() * osg::Vec3f(0,1,0),
+                    static_cast<SceneUtil::PositionAttitudeTransform*>(blocker.getRefData().getBaseNode())->getAttitude() * osg::Vec3f(0,1,0),
```

Comment:
These static casts are really bad. Probably it would be better to have the `getBaseTransform()` method (which should throw an exception if the value is nullptr).